internal/librarian/command.go

```diff
@@ -215,7 +215,14 @@ func createWorkRoot(t time.Time, workRootOverride string) (string, error) {
 }
 
 // No commit is made if there are no file modifications.
-func commitAll(repo *gitrepo.Repository, msg, userName, userEmail string) error {
+func commitAll(repo *gitrepo.Repository, msg, pushConfig string) error {
+	parts := strings.Split(pushConfig, ":")
+	if len(parts) != 2 {
```

Comment:
There is a [IsValid()](https://github.com/googleapis/librarian/blob/24456520e0a9ea2a8f52af64f79e80992261dbb4/internal/config/config.go#L376) function in config.go. I wonder if we should do validation on format there, so we could potentially return error sooner?

Reply:
I added validation here and I kept the validation in `commitAll()`, because I found `isValid()` is not used anywhere other than tests. Please let me know if I missed its usage.